Earthfile

```diff
@@ -94,7 +94,7 @@ earth:
 	ARG EARTHLY_TARGET_TAG
 	ARG VERSION=$EARTHLY_TARGET_TAG
 	ARG DEFAULT_BUILDKITD_IMAGE=earthly/buildkitd:$VERSION
-	ARG DEFAULT_DEBUGGER_IMAGE=earthly/debugger:$VERSION
+	ARG DEFAULT_DEBUGGER_IMAGE=earthly/debugger-docker-image:asdf@sha256:b3d0c0db9b08c3e63b105f6efb9bd028632955d337128188406c0256d95f39b2
```

Comment:
Hmm.. it doesn't look quite right. The image is `earthly/debugger`, right?

Reply:
that's embarrassing, I don't know how that happened. it should have been `debugger:docker-image@...`. I fixed this now.